README.md

```diff
@@ -51,11 +48,14 @@ for agent in env.agent_iter():
 
     env.step(action)
 env.close()
+
+# optionally, you can scalarize the reward with weights
+env = momaland.LinearReward(env, weight=np.array([0.6, 0.2, 0.2]))
```

Comment:
Aren't the weights supposed to be dictionary in our case?

Reply:
leftover copy pasta from mogym